python/ray/tune/hyperband.py

```diff
@@ -367,11 +383,11 @@ def _calculate_total_work(self, n, r, s):
 
     def __repr__(self):
         status = ", ".join([
-            "n={}".format(self._n),
-            "r={}".format(self._r),
-            "completed={}%".format(int(100 * self.completion_percentage()))
+            "Max Size (n)={}".format(self._n),
+            "Milestone (r)={}".format(self._r),
+            "completed={:.1%}".format(self.completion_percentage())
             ])
-        counts = collections.Counter()
-        for t in self._all_trials:
-            counts[t.status] += 1
-        return "Bracket({}): {}".format(status, dict(counts))
+        counts = collections.Counter([t.status for t in self._all_trials])
+        trial_statuses = ", ".join(sorted(
+            ["{}: {}".format(k, v) for k, v in counts.items()]))
+        return "Bracket({}): {{{}}} ".format(status, trial_statuses)
```

Comment:
Is this supposed to have so many nested brackets?

Reply:
yeah; {{ }} is for regular brackets and {} is for format placeholding